pkg/cli/whoami/whoami.go

```
@@ -181,7 +183,14 @@ func (o *WhoAmIOptions) getWebConsoleUrl() (string, error) {
 func (o *WhoAmIOptions) Run() error {
 	switch {
 	case o.ShowToken:
-		fmt.Fprintf(o.Out, "%s\n", o.ClientConfig.BearerToken)
+		token, err := currentBearerToken(o.ClientConfig)
+		if err != nil {
+			return err
+		}
+		if len(token) == 0 {
+			return fmt.Errorf("no token is currently in use for this session")
+		}
+		fmt.Fprintf(o.Out, "%s\n", token)
```

golangci-lint (2.12.2)

[error] Error return value of `fmt.Fprintf` is not checked

(errcheck)